server/services/config/crm7DisplayConfig.json

```diff
@@ -276,6 +279,9 @@
               "label": "Please submit the indictment with this claim. Please indicate whether you will attach it electronically or if you are submitting it by Post / DX.",
               "apiField": "caseDetails.indictmentAttachment"
             },
+            {
+              "subHeading": "Please submit the indictment with this claim. Please indicate whether you will attach it electronically or if you are submitting it by Post / DX."
```

Comment:
Is this heading correct, as it the same as the label above:- 

`              "label": "Please submit the indictment with this claim. Please indicate whether you will attach it electronically or if you are submitting it by Post / DX.",`

Reply:
Yes, you're right but it wasn't displayed.